client-src/src/pages/inq-form1/inq-form1.ts

```diff
@@ -94,7 +114,13 @@ export class InqForm1Page {
           }
         },
         error => { console.log("POST unsuccessful, the server returned this error:", error); this.loading.dismissAll(); },
-        () => { console.log("complete"); this.loading.dismissAll(); }
+        () => {
+          console.log("complete");
+          this.loading.dismissAll();
+          if(this.responseData.data){
+            this.navCtrl.push(InqForm2Page,{ data: this.responseData });
+          }
+        }
         );
     }else{
       this.notify.showError("Invalid Form! Please fill proper values");
```

Comment:
Create and move to NotificationMessageService

Reply:
Addressed in commit 085b3c9